utils/systemd/daos_server.service

```diff
@@ -13,12 +13,24 @@ RuntimeDirectoryMode=0755
 ExecStart=/usr/bin/daos_server start
 StandardOutput=journal
 StandardError=journal
+StartLimitBurst=5
```

Comment:
What situation do you want to address with this setting ?? Do I correctly understand that this means we will tolerate up to 5 service restarts within StartLimitIntervalSec/DefaultStartLimitIntervalSec=10s before to decide stopping restart attempts ?? Well, my experience with daos_server service startup failures is that there is something wrong that needs to be investigated, so I would better set StartLimitBurst=1 and raise a little bit StartLimitIntervalSec to something like 120s. But I am sure we will had to schedule a specific meeting about this topic where everybody can have a different idea !!

Reply:
I only moved this one around and did not change the value. Not sure who added it. @mjmac any idea?

Reply:
Hmm. Well, git blame shows that @dpquigl added that line. It looks like `StartLimitIntervalSec` is set to 60s up in the unit block above this, so this is saying that `daos_server` can only attempt to restart 5 times within 60s, if I understand correctly.

I do tend to agree that there probably isn't much value in having the service attempt to restart multiple times, as errors which prevent successful startup (bad config, missing hardware, etc) aren't likely to resolve themselves.

@dpquigl: Was there a specific rationale behind choosing that value? Should we just revert to the default?

Reply:
From my side, everything seems fine at the exception of this parameter `StartLimitIntervalSec `. Is there any feedback on the motivation to choose this value ?

Reply:
I believe this was set the way it was because without it we were flooding the logs with error messages if DAOS wasn't configured correctly and systemd had it enabled.